awswrangler/_arrow.py

```diff
@@ -76,6 +76,9 @@ def _table_to_df(
     if table.schema.metadata is not None and b"pandas" in table.schema.metadata:
         metadata = json.loads(table.schema.metadata[b"pandas"])
 
+    if "schema" in kwargs:
+        kwargs.pop("schema")
```

Comment:
There's a bit of a mess between different pyarrow calls regarding which arguments are supported. I'm opening this draft PR to show a couple of examples, like `schema` or `coerce_int96_timestamp_unit` that we handle as a separate arg